jupyter_server_proxy/standalone/__init__.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import annotations  # For Python 3.8 compatibility
+
+import argparse
+import logging
+import os
+from urllib.parse import urlparse
+
+from tornado import ioloop
+from tornado.httpserver import HTTPServer
+from tornado.log import app_log as log
+from tornado.log import enable_pretty_logging, gen_log
+
+from .activity import start_activity_update
+from .proxy import configure_ssl, make_proxy_app
+
+
+def _default_address_and_port() -> tuple[str, int]:
```

Comment:
jupyter-server-proxy officially supports Python >= 3.8. But while that type annotation only works since 3.9, we can use `from __future__ import annotations` to enable it in older versions (back until 3.7). But I will make the comment more specific.